lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
This wasn't about const/let, but why we want an object with the function instead of just having a single function that we import. I get that it holds state, but what I did not get what that this can help with option parsing and encapsulating the configuration inside the object. Still, the class name is odd 😛